src/server.cpp

```diff
@@ -2668,7 +2668,8 @@ void Server::sendRequestedMedia(session_t peer_id,
 	assert(client);
 
 	infostream << "Server::sendRequestedMedia(): Sending "
-		<< tosend.size() << " files to " << client->getName() << std::endl;
+		<< tosend.size() << " files to " << client->getName()
+		<< ", peer_id=" << peer_id << std::endl;
```

Comment:
which player has which peer id can be inferred from the logs so I think it's not essential.
(applies to this entire function)

Reply:
Done in d0044bf27